src/uncss.js

```diff
@@ -218,18 +218,12 @@ function init(files, options, callback) {
         timeout: 0,
         report: false,
         ignoreSheets: [],
+        html: files,
         // gulp-uncss parameters:
         raw: null
     });
 
-    return promise
-        .using(phantom.init(options.phantom), function () {
-            return getHTML(files, options)
-                .spread(getStylesheets)
-                .spread(getCSS)
-                .spread(processWithTextApi);
-        })
-        .asCallback(callback, { spread: true });
+    serializedQueue.push(options, callback);

```

Comment:
@mikelambert Shouldn't this return a promise as well? (See JSDocs above.) This would also get the Travis build passing.